dask_sql/server/responses.py

```diff
@@ -65,13 +65,18 @@ def __init__(self, request: Request, next_url: str = None, cancel_url: str = Non
 class DataResults(QueryResults):
     @staticmethod
     def get_column_description(df):
-        sql_types = [str(python_to_sql_type(t)) for t in df.dtypes]
+        sql_types = [str(python_to_sql_type(t)).lower() for t in df.dtypes]
         column_names = df.columns
         return [
             {
                 "name": column_name,
-                "type": sql_type.lower(),
-                "typeSignature": {"rawType": sql_type.lower(), "arguments": []},
+                "type": sql_type,
+                "typeSignature": {
+                    "rawType": sql_type,
+                    "arguments": []
+                    if sql_type not in ("char", "varchar")
+                    else [{"type": "LONG", "value": 10}],
```

Comment:
Hardcoded this for now as at least on DBeaver it doesn't seem to check the column data against this value; in reality we would ideally want to expose this information through the SQL type and use that instead